### PR TITLE
fix image broken

### PR DESCRIPTION
**Pull request by Damoness**

### CODE REVIEW

File: files/zh-cn/web/css/object-fit/index.md [hunk 1]
```diff
@@ -36,33 +36,23 @@ object-fit 属性由下列的值中的单独一个关键字来指定。
...
 ```html
-<div>
+<section>
   <h2>object-fit: fill</h2>
-  <img src="https://mdn.mozillademos.org/files/6457/mdn_logo_only_color.png" alt="MDN Logo" class="fill"/>
-
-  <img src="https://mdn.mozillademos.org/files/6457/mdn_logo_only_color.png" alt="MDN Logo" class="fill narrow"/>
-
+  <img class="fill" src="mdn_logo_only_color.png" alt="MDN Logo">
+  <img class="fill narrow" src="mdn_logo_only_color.png" alt="MDN Logo">
```
Comment (yin1999) [inline files/zh-cn/web/css/object-fit/index.md:42]:
```suggestion
  <img class="fill" src="/en-US/docs/Web/CSS/object-fit/mdn_logo_only_color.png" alt="MDN Logo">
  <img class="fill narrow" src="/en-US/docs/Web/CSS/object-fit/mdn_logo_only_color.png" alt="MDN Logo">
```

File: files/zh-cn/web/css/object-fit/index.md [hunk 2]
```diff
@@ -36,33 +36,23 @@ object-fit 属性由下列的值中的单独一个关键字来指定。
...
   <h2>object-fit: contain</h2>
-  <img src="https://mdn.mozillademos.org/files/6457/mdn_logo_only_color.png" alt="MDN Logo" class="contain"/>
-
-  <img src="https://mdn.mozillademos.org/files/6457/mdn_logo_only_color.png" alt="MDN Logo" class="contain narrow"/>
-
+  <img class="contain" src="mdn_logo_only_color.png" alt="MDN Logo">
+  <img class="contain narrow" src="mdn_logo_only_color.png" alt="MDN Logo">
```
Comment (yin1999) [inline files/zh-cn/web/css/object-fit/index.md:45]: And just follow above

**Review by yin1999 — COMMENTED** [review]
Hi @Damoness, this is a bit annoying us for we can't fallback to en-US when the image does not exist in l10n (see: #3752). To work around this, can you reference to the image from en-US?

Then, we can remove the picture in l10n folder

File: files/zh-cn/web/css/object-fit/index.md (hunk 2 above)
Comment (Damoness) [reply files/zh-cn/web/css/object-fit/index.md]: thanks

**Review by yin1999 — APPROVED** [review]
Thank you

**Pull request merged**